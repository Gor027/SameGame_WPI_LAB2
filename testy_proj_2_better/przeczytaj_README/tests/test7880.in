 @ @K@@ K  KK K  K@K  K @K@@KKKKK@KKK@K @ KK K@K@K@@@K @K@K@@ KK @@@@@@@@@   @K@
  K@KK K  KKK@KK KK@@ @KK@@K @  @@ @
K KK@@ @@@@K @  K@K K  KK@K   @ K @ K   @ @    K@@ KK
  KK@ @K  @@K  @K@@K @@K@K
