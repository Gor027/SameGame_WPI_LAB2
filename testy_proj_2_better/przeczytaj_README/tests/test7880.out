  @K      K     K@     @K
 K@K@ K  KKK    KKK  KKK@@     K
 KK@K@@K @KK@KKK@K@@ @KK@@@ KK @@KKK  @K   K@K @@
@KK@@@K@@@@K@K@K@@KK@@K@KKKK@@KK@@@K@K@@KK@K@@@KKK@K@K@@KK@@@@@@@@@@K@
